### ISSUE TITLE
Ubuntu 14.04 LTS Issue with apparmor removal

### ISSUE THREAD

**Issue by Caffe1neAdd1ct** [sentora/sentora-installers#66]
Hi Guys,

Noticed an issue, when running the sentora installer it runs:

`apt-get remove -y --purge apparmor* &> /dev/null`

Actually results in:

```
The following packages will be REMOVED
  dbus* debhelper* dh-apparmor* libapparmor1* libpam-systemd* pkg-php-tools*
```

This removes alot more than just apparmor and causes huge dep issues with apt-get later, i would suggest changing:

```
        echo -e "\n-- Disabling and removing AppArmor, please wait..."
        /etc/init.d/apparmor stop &> /dev/null
        update-rc.d -f apparmor remove &> /dev/null
        apt-get remove -y --purge apparmor* &> /dev/null
        disable_file /etc/init.d/apparmor &> /dev/null
        echo -e "AppArmor has been removed."
```

to

```
        echo -e "\n-- Disabling and removing AppArmor, please wait..."
        /etc/init.d/apparmor kill &> /dev/null
        update-rc.d -f apparmor remove
        apt-get remove -y --purge apparmor apparmor-utils
        echo -e "AppArmor has been removed."
```

Wildcard removal of packages should be avoided.

**Comment by 5050** [comment]
Adopted ! (commit 305cc3d4b2)

**Comment by Caffe1neAdd1ct** [comment]
@5050 Cheers :+1: 

Hoping for some free time soon, i'll be back on the forums soon :)

**Comment by Caffe1neAdd1ct** [comment]
Quick note on how to fix this for existing Ubuntu 14.04 installations:

 * Re-install system packages: `apt-get update; apt-get upgrade; apt-get install libapparmor1 libpam-systemd systemd-services dbus`
 * Restart the server `shutdown -r now`
 * Remove apparmor the new way: `/etc/init.d/apparmor kill &> /dev/null; update-rc.d -f apparmor remove; apt-get remove -y --purge apparmor apparmor-utils`
 * Upgrade again should now work `apt-get update; apt-get upgrade`
